http-client/src/main/java/io/airlift/http/client/jetty/HttpClientLogLayout.java

```diff
@@ -0,0 +1,35 @@
+package io.airlift.http.client.jetty;
+
+import ch.qos.logback.core.LayoutBase;
+
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+import static java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME;
+
+class HttpClientLogLayout
+        extends LayoutBase<HttpRequestEvent>
+{
+    private static final DateTimeFormatter ISO_FORMATTER = ISO_OFFSET_DATE_TIME.withZone(ZoneId.systemDefault());
+
+    @Override
+    public String doLayout(HttpRequestEvent event)
+    {
+        StringBuilder builder = new StringBuilder();
+        builder.append(ISO_FORMATTER.format(event.getTimeStamp()))
+                .append('\t')
+                .append(event.getProtocol())
+                .append('\t')
+                .append(event.getMethod())
+                .append('\t')
+                .append(event.getRequestUri()) // TODO: escape
+                .append('\t')
+                .append(event.getResponseCode())
+                .append('\t')
+                .append(event.getResponseSize())
+                .append('\t')
+                .append(event.getTimeToLastByte())
+                .append('\n');
```

Comment:
should we add trace token?  It is passed as a header, so it should be available when constructing the event.